### PR TITLE
[skin.estuary] Add Hebrew and Armenian typefaces to the fonts

### PR DESCRIPTION
**Pull request by arthur-liberman**
## Description
This PR adds Hebrew and Armenian typefaces to the default Estuary fonts.
NotoSans is available in multiple languages directly from Google, and that was my source. The added characters had to be scaled by 200% to be in the correct size.

I used Heebo and Mardoto fonts to add Hebrew and Armenian to the Roboto-Thin font. These are fonts based on Roboto, but with Hebrew and Armenian typefaces.

The new typefaces were merged into the existing NotoSans TTF files from the Kodi repository, so only the missing characters were added, and none of the original ones were removed or replaced.
Roboto was updated to the latest available build of the font. Then I merged the existing Roboto-Thin font, followed by Heebo and Mardoto.

## Motivation and Context
NotoSans looks much better than Arial, but it lacked Hebrew and Armenian typefaces, which resulted in blank characters. 
In addition, installing Hebrew UI without changing the skin font to Arial first, would result in an unusable UI, as all labels would simply be blank squares. 

Armenian typefaces were added too, as some of my TV channels are Armenian, and it's nice to see the actual labels, even if you can't read it.

The fonts can be easily extended with other languages too in the future.

## How Has This Been Tested?
Tested on x64 Kodi nightly and Hebrew UI and Armenian texts.

## Screenshots (if appropriate):
Hebrew:
image
image
Armenian (TV window)
image

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [X] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by ronie** [comment]
thx!
tested and works ok on my end.

File: addons/skin.estuary/fonts/heebo_licence.txt [hunk 1]
```diff
@@ -0,0 +1,97 @@
+Copyright (c) <dates>, <Copyright Holder> (<URL|email>),
+with Reserved Font Name <Reserved Font Name>.
+Copyright (c) <dates>, <additional Copyright Holder> (<URL|email>),
+with Reserved Font Name <additional Reserved Font Name>.
+Copyright (c) <dates>, <additional Copyright Holder> (<URL|email>).
```
[the remark this answers is not in the document]
Comment (ronie) [reply addons/skin.estuary/fonts/heebo_licence.txt]: @arthur-liberman heebo license can be found here:
https://github.com/OdedEzer/heebo/blob/master/OFL.txt

[the remark this answers is not in the document]
Comment (arthur-liberman) [reply addons/skin.estuary/fonts/heebo_licence.txt]: Sure. I wasn't sure how to properly fill those fields.

[the remark this answers is not in the document]
Comment (arthur-liberman) [reply addons/skin.estuary/fonts/heebo_licence.txt]: @ronie thanks, I'll update the PR after work.

[the remark this answers is not in the document]
Comment (arthur-liberman) [reply addons/skin.estuary/fonts/heebo_licence.txt]: @wsnipex Fixed.

**Comment by MartijnKaijser** [comment]
@ronie good for merge?

**Comment by ronie** [comment]
yup go for it

**Pull request merged**